cognite/client/data_classes/iam.py

```diff
@@ -336,23 +336,27 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         }
 
 
+SessionStatus: TypeAlias = Literal["ready", "active", "cancelled", "expired", "revoked", "access_lost"]
```

Comment:
I think we should upper case these:

```suggestion
SessionStatus: TypeAlias = Literal["READY", "ACTIVE", "CANCELLED", "EXPIRED", "REVOKED", "ACCESS_LOST"]
```